guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Creating_an_Image_for_Amazon_EC2_{context}"]
+= Creating an Image for Amazon EC2
+
+You can create images for Amazon EC2 from within {Project}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *Compute Resources*.
+. Select your Amazon EC2 provider.
+. Click *Create Image*.
+
+* In the *Name* field, enter a meaningful and unique name for your EC2 image.
+* From the *Operating System* list, select an operating system to associate with the image.
+* From the *Architecture* list, select an architecture to associate with the image.
+* Enter the *Username* needed to SSH into the machine.
+* Enter the *Image ID* provided by Amazon or a operating system vendor.
```

Comment:
```suggestion
* In the *Username* field, enter the username needed to SSH into the machine.
* In the *Image ID* field, enter the image ID provided by Amazon or a operating system vendor.
```
I assume these are text fields. In that case, this syntax would be consistent with the first step.